cmd/kubeadm/app/apis/output/v1alpha2/types.go

```diff
@@ -82,3 +84,45 @@ type UpgradePlan struct {
 
 	ConfigVersions []ComponentConfigVersionState `json:"configVersions"`
 }
+
+// Certificate represents information about check-expiration for one certificate
+type Certificate struct {
```

Comment:
I'm not sure which type for these fields is more suitable for machine. 

- ExpirationDate: string or metav1.Time
- ResidualTime: time.Duration or string
- ExternallyManaged: bool or string (yes/no)

can you help me to choose the right type for these fields? @neolit123

Reply:
- ExpirationDate: string or metav1.Time

metav1.Time if possible.

- ResidualTime: time.Duration or string

metav1.Duration

- ExternallyManaged: bool or string (yes/no)

bool

Reply:
also ExternallyManaged -> IsExternallyManaged

Reply:
added a new version and the PR description is updated. the text print hasn't been changed.

@neolit123 please review again.